src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.cs

```diff
@@ -72,7 +72,7 @@ internal partial struct TarHeader
 
         // PAX attributes
 
-        internal Dictionary<string, string> _extendedAttributes;
+        internal Dictionary<string, string>? _extendedAttributes;
```

Comment:
Made this nullable because it is _only_ required in PAX, and we always have to check in that format if there is a valid dictionary instance before consuming it.